Content.Shared/IdentityManagement/SharedIdentitySystem.cs

```diff
@@ -39,4 +45,28 @@ private void OnMaskToggled(Entity<IdentityBlockerComponent> ent, ref ItemMaskTog
     {
         ent.Comp.Enabled = !args.IsToggled;
     }
+
+    /// <summary>
+    ///     When the identity of a person is changed, searches for a criminal records console and compares the name of
+    ///     the new identity with a dictionary stored on the console. If the new name has a criminal status attached to it,
+    ///     the person will get the same criminal status until they change identity again.
+    /// </summary>
+    private void OnIdentityChanged(Entity<IdentityComponent> ent, ref IdentityChangedEvent args)
+    {
+        var query = EntityQueryEnumerator<CriminalRecordsConsoleComponent>();
+        var name = Identity.Name(ent, _entityManager);
+        while (query.MoveNext(out var uid, out var criminalRecordsConsole))
+        {
+            if (criminalRecordsConsole.Criminals.TryGetValue(name, out var criminal))
+                _criminalRecordsConsole.SetCriminalIcon(name, criminal,ent);
+            else
+                RemComp<CriminalRecordComponent>(ent);
+            break;
```

Comment:
there can be multiple consoles do not break

Reply:
I break because I currently update all consoles at the same time, so they all should have the same list of names.

Reply:
if they are on different stations they wont

Reply:
I'm not storing the value on the consoles anymore